tools/sof-kernel-log-check.sh

```diff
@@ -419,6 +419,18 @@ ignore_str="$ignore_str"'|nvme0: Admin Cmd\(0x[[:digit:]]+\), I/O Error \(sct 0x
 # SDW related logs
 #
 
+# This expects an array like for instance:
+#    sof_local_extra_kernel_ignores=(-e 'error 1' -e err2)
+# Arrays
+# - provide whitespace/quoting safety
+# - can be empty/optional
+# - can be arbitrarily complex. Best avoided but only on specific systems anyway.
+# shellcheck disable=SC2154
+if &>/dev/null declare -p sof_local_extra_kernel_ignores; then
+    dlogw "Ignoring extra errors on this particular system:"
```

Comment:
One concern I have with this facility is that when adding local exceptions becomes easier, it can be used too much and people reading the test logs might not realize some DUTs have local exceptions others don't have. The global ignore list has drawbacks, but at least one needs to make a git commit with proper description and leave something that can be search through git history. OTOH, this dlogw does make local exception use clear in the log, which alleviates my concern. It really depends on how this will be used.